lib/plugins/door43translation/lang/getLanguageNames.php

```diff
@@ -1,50 +0,0 @@
-<?php
```

Comment:
This file was removed because we are no longer generating the langnames.txt file.